product_brand/views/product_brand_view.xml

```diff
@@ -83,29 +83,19 @@
                 <field name="description"/>
                 <templates>
                     <t t-name="kanban-box">
-                        <div class="oe_kanban_vignette oe_semantic_html_override">
-                            <a type="open">
-                                <img t-att-src="kanban_image('product.brand', 'logo', record.id.raw_value)"
-                                     class="oe_kanban_image"/>
-                            </a>
+                        <div class="oe_kanban_global_click">
+                            <div class="o_kanban_image">
+                                <img t-att-src="kanban_image('product.brand', 'logo', record.id.raw_value)"/>
+                            </div>
                             <div class="oe_kanban_details">
                                 <h4>
-                                    <a type="open">
-                                        <field name="name"/>
-                                    </a>
+                                    <field name="name"/>
                                 </h4>
                                 <div>
-                                    <a name="%(product_brand.action_open_brand_products)d"
-                                       type="action">
+                                    <a name="%(product_brand.action_open_brand_products)d" type="action">
                                         <t t-esc="record.products_count.value"/> Products
                                     </a>
                                 </div>
-                                <span>
-                                    <t t-esc="record.description.value.substr(0,200)"/>
```

Comment:
Why removing the description?

Reply:
Because this makes the kanban view too dense.

We usually don't see a description inside a kanban card. Not in tasks, nor partners, nor products...

Reply:
OK, I agree